src/plibsysconfig.h.in

```diff
@@ -69,6 +70,16 @@ P_BEGIN_DECLS
 #  define P_BYTE_ORDER P_LITTLE_ENDIAN
 #endif
 
+#define p_va_list va_list
```

Comment:
p_va_copy is compiler dependent - and in fact using <stdarg.h> is a major choice for a developper because it is fundamentally incompatible with the other choice that is<vardargs.h> - thus ok for you if I use a configure_file( pstdarg.h.in pstdarg.h) ?
Obviously pstdarg.h.in would include stdarg.h -;

Reply:
We can implement something in between. Leave only the macro PLIBSYS_VA_COPY in plibsysconfig.h file. This file is always included before any other. It could be useful for a user in some cases to know whether a system has va_copy native support or not.
In a separate file pstdarg.h you can define all the p_va_* macros implementation, also based on the definition of PLIBSYS_VA_COPY for p_va_copy(). Do not forget to include pstdarg.h file in plibsys.h.
This way we do not need additional configuration file, but rather based on configure process output select a proper implementation.

Reply:
Done.
In the meantime I was re-reading forums and C specs, and, despite I am almost sure that the ``memcpy`` hack is ok when there is no implementation of va_copy because this happens /only/ with old versions of MSVC as far as I know (and with MSVC a va_list is a pointer since the beginning of this compiler, va_end just putting the value to 0), the standard definitely does not guarantees this can always work.

So two cases:
- this is ok on all tested platforms, a more in-depth test can prove that, you're right.
- I am thinking to an alternative like ``struct p_va_list { va_list _native /* first member is not a hasard */; short _memcopied; }``

Will keep you informed.

Reply:
Agree